parsons/ngpvan/saved_lists.py

```diff
@@ -64,6 +67,82 @@ def download_saved_list(self, saved_list_id):
         else:
             return Table.from_csv(job['downloadUrl'])
 
+    def upload_saved_list(self, tbl, list_name, folder_id, url_type, id_type='vanid', replace=False,
+                          **url_args):
+        """
+        Upload a saved list. Invalid or unmatched person id records will be ignored. Your api user
+        must be shared on the target folder.
+
+        `Args:`
+            tbl: parsons.Table
+                A parsons table object containing one column of person ids.
+            list_name: str
+                The saved list name.
+            folder_id: int
+                The folder id where the list will be stored.
+            url_post_type: str
+                The cloud file storage to use to post the file. Currently only ``S3``.
+            id_type: str
+                The primary key type. The options, beyond ``vanid`` are specific to your
+                instance of VAN.
+            replace: boolean
+                Replace saved list if already exists.
+            **url_kwargs: kwargs
+                Arguments to configure your cloud storage url type.
+                    * S3 requires ``bucket`` argument and, if not stored as env variables
+                      ``aws_access_key`` and ``aws_secret_access_key``.
+        `Returns:`
+            dict
+                Upload results information included the number of matched and saved
+                records in your list.
+        """
+
+        # Move to cloud storage
+        file_name = str(uuid.uuid1()) + '.zip'
+        public_url = cloud_storage.post_file(tbl, url_type, file_path=file_name, **url_args)
+        csv_name = files.extract_file_name(file_name, include_suffix=False) + '.csv'
```

Comment:
Minor thing, but if you just have a temp variable set to `uuid.uuid1()`, you could use it to build the zip and csv filename and wouldn't need to extract here.